app/views/uploads/_data_criteria.html.erb

```diff
@@ -1,127 +1,124 @@
-            <% if (dataCriteria[:derivation_operator] == 'INTERSECT' ||dataCriteria[:derivation_operator] == 'UNION')%>
-              <% if !expandVariable %>
-                <li class="set-op-item">
-              <% else %>
-                <li>
-              <% end %>
+    <% elsif data_criteria[:children_criteria] %>
+      <% if data_criteria[:derivation_operator] == 'INTERSECT' ||
+            data_criteria[:derivation_operator] == 'UNION' &&
+            !expand_variable %>
+        <ul><li><span class="set-op rationale-target">
+          <%= data_criteria[:derivation_operator] %> of:
+        </span></li>
+      <% end %>
+      <ul>
+        <% data_criteria[:children_criteria].each do |child| %>
+          <% if (data_criteria[:derivation_operator] == 'INTERSECT' ||data_criteria[:derivation_operator] == 'UNION') %>
+            <% if !expand_variable %>
+              <li class="set-op-item">
             <% else %>
               <li>
-                <span class="conjunction {{../../dataCriteria.key}} rationale-target"><%= (dataCriteria[:derivation_operator] == 'XPRODUCT')? 'AND':'' %>: <span class="sr-only sr-highlight-status"></span></span>
-            <% end %>
-              <%= render partial: 'data_criteria', locals: {reference: child, expandVariable: false, measure:measure, tag: "span"} %>
-            </li>
-          <% end %>
-          <% if dataCriteria[:temporal_references] %>
-            <% dataCriteria[:temporal_references].each do |temp_ref| %>
-              <li>
-                <span class="{{../dataCriteria.key}} rationale-target">
-                  <%= render partial: 'temporal_reference', locals: {temporalReference: temp_ref, measure:measure, tag: "span"} %>
-                </span>
-              </li>
             <% end %>
+          <% else %>
+            <li>
+              <span class="conjunction rationale-target"><%= (data_criteria[:derivation_operator] == 'XPRODUCT')? 'AND':'' %>: <span class="sr-only sr-highlight-status"></span></span>
           <% end %>
-        </ul>
-        <% if dataCriteria[:derivation_operator] == 'INTERSECT' || dataCriteria[:derivation_operator] == 'UNION'%>
-          <% unless expandVariable %>
-            </ul>
+            <%= render partial: 'data_criteria', locals: {reference: child, expand_variable: false, measure:measure, tag: "span"} %>
+          </li>
+        <% end %>
+        <% if data_criteria[:temporal_references] %>
+          <% data_criteria[:temporal_references].each do |temp_ref| %>
+            <li>
+              <span class="rationale-target">
+                <%= render partial: 'temporal_reference', locals: {temporal_reference: temp_ref, measure:measure, tag: "span"} %>
+              </span>
+            </li>
           <% end %>
         <% end %>
-
+      </ul>
+      <% if data_criteria[:derivation_operator] == 'INTERSECT' ||
+            data_criteria[:derivation_operator] == 'UNION' &&
+            !expand_variable %>
```

Comment:
This 3-part conditional is in two places. Should just set to a variable so that it's only in one place.